src/test/java/com/Searchutil/app/SearchServiceUtilTest.java

```diff
@@ -0,0 +1,49 @@
+package com.Searchutil.app;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import static org.mockito.Mockito.doThrow;
+
+import java.io.IOException;
+
+import org.apache.solr.client.solrj.SolrClient;
+import org.apache.solr.client.solrj.impl.HttpSolrClient;
+import com.searchservice.app.domain.utils.SearchServiceUtil;
+
+public class SearchServiceUtilTest {
+  
+	HttpSolrClient searchServiceClient = new HttpSolrClient.Builder("http://localhost:8983/solr/collection1").build();
+	SolrClient client = new HttpSolrClient.Builder("http://localhost:8983/solr/collection1").build();
```

Comment:
We're not supposed to call solr in Unit tests, so don't provide actual url here.